tklinenums/tklinenums.py

```diff
@@ -298,15 +315,16 @@ def reload(self) -> None:
     root = Tk()
 
     style = Style()
-    style.configure("TLineNumbers", background="#ffffff", foreground="#2197db")
+    style.configure("TLineNumbers", background="#ffffff")
 
     text = Text(root)
     text.pack(side="right")
 
     for i in range(50):
         text.insert("end", f"Line {i+1}\n")
 
-    linenums = TkLineNumbers(root, text)
+    linenums = TkLineNumbers(root, text, ttk_theme=True)
+    linenums.set_colors(foreground="#2197db")
```

Comment:
```suggestion
    linenums = TkLineNumbers(root, text, color_provider=ttk_theme_color_provider)
```

And somewhere else in the file:

```
def ttk_theme_color_provider() -> tuple[str, str]:
    fg = self.tk.eval("ttk::style lookup TLineNumbers -foreground")
    bg = self.tk.eval("ttk::style lookup TLineNumbers -background")
    return (fg, bg)
```